rt/src/core/rt/misc.volt

```diff
@@ -86,10 +86,10 @@ fn vrt_encode_static_u8(ref buf: char[6], c: dchar) size_t;
  *
  * `index` is updated to the next codepoint's start position.
  */
-fn vrt_decode_u8_d(str: string, ref index: size_t) dchar;
+fn vrt_decode_u8_d(str: const(char)[], ref index: size_t) dchar;
```

Comment:
These should be `SinkArg` or `scope const(char)[]` as they do not keep a reference to the memory (or at least shouldn't).